src/stores/events/EventsDataStore.ts

```diff
@@ -92,10 +88,19 @@ export default class EventsDataStore {
 	@computed
 	public get targetNodePath() {
 		if (!this.targetNode) return null;
-		const parentIds = this.eventStore
-			.getParentNodes(this.targetNode.eventId, this.eventsCache)
-			.map(parentEventNode => parentEventNode.eventId);
-		return [...parentIds, this.targetNode.eventId];
+
```

Comment:
is there a reason it had to be rewritten? looks like it reimplements `eventStore.getParentNodes`.

Reply:
Here we are accumulating event by id they was searched. Cache contains unknown group if event wasn't found, but we need real ids